src/main/java/com/telefonica/iot/perseo/Configuration.java

```diff
@@ -67,7 +69,7 @@ public static synchronized boolean reload() {
         InputStream stream;
         String defaultMaxAge;
         String defaultURL;
-        String actionPath = "/actions/do";
+        String actionPath = ResourceBundle.getBundle("perseo-core").getString("action.path");
```

Comment:
Not sure of understanding this change... where "action.path" configuration is located in the resources of the proyect? Some .properties file? Which one?

Reply:
This is this is due to an issue of the portability axis (URIs should not be hardcoded).

"action.path" as been located in perseo-core.properties file.

References:
https://wiki.sei.cmu.edu/confluence/display/java/MSC03-J.+Never+hard+code+sensitive+information

Reply:
NTC